core/matching/orderbook.go

```diff
@@ -704,6 +704,26 @@ func (b *OrderBook) RemoveOrder(id string) (*types.Order, error) {
 	return order, nil
 }
 
+func (b *OrderBook) CancelOrdersOnReject() []*types.Order {
+	buys, sells := b.getSide(types.SideBuy).getLevels(), b.getSide(types.SideSell).getLevels()
+	ret := []*types.Order{}
+	for _, lvl := range buys {
+		for _, o := range lvl.orders {
+			o, _ = b.DeleteOrder(o)
+			o.Status = types.OrderStatusCancelled
```

Comment:
I don't think CANCELLED is thr correct status. Both in the spec at the protos definitions, CANCELLED is related to a user actively cancelling their order and STOPPED is for when the network makes that decision:
```
    // Used for orders cancelled by the party that created the order
    STATUS_CANCELLED = 3;
    // Used for unfilled FOK or IOC orders, and for orders that were stopped by the network
    STATUS_STOPPED = 4;
```

also the orderbook has the `Settled()` method cleans up orders already, so I don't think we need two ways to clean up all orders and set a particular status.

Reply:
stopped would be the correct